coderd/provisionerkey/provisionerkey.go

```diff
@@ -21,11 +21,16 @@ func New(organizationID uuid.UUID, name string) (database.InsertProvisionerKeyPa
 	hashedSecret := sha256.Sum256([]byte(secret))
 	token := fmt.Sprintf("%s:%s", id, secret)
 
+	if tags == nil {
+		tags = map[string]string{}
+	}
```

Comment:
Not in this PR, but we should make sure the `StringMap` never marshals as a `null`